system-tests/test/session_spec.ts

```diff
@@ -31,7 +31,7 @@ const onServer = function (app) {
   })
 
   app.get('/cross_origin_iframe/:name', (req, res) => {
-    res.send(`<html><body><h1>cross_origin_iframe ${req.params.name}</h1><iframe src="https://127.0.0.2:44665/set-localStorage/${req.params.name}"</body></html>`)
+    res.send(`<html><body><h1>cross_origin_iframe ${req.params.name}</h1><iframe src="https://127.0.0.1:44665/set-localStorage/${req.params.name}"</body></html>`)
```

Comment:
Why did these change from https://127.0.0.2 to https://127.0.0.1?

Reply:
127.0.0.2 doesn't loopback by default on a Mac, but it does in Linux. So these were failing when run locally on a Mac. It's likely no one had tried running them on a Mac before and, of course, they pass when run in CI. We didn't see any need to use 127.0.0.2 for them, so we changed it so that they would work regardless of OS.